x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/columns.tsx

```diff
@@ -240,4 +243,25 @@ export function useMonitorListColumns({
       ],
     },
   ];
+
+  if (!canEditSynthetics) {
+    // replace last column with a tooltip
+    columns[columns.length - 1] = {
```

Comment:
nit: Replacing by index may be fragile than conditionally switching the column in place.